src/SingleEventManager.lua

```diff
@@ -53,7 +53,14 @@ function SingleEventManager:connectEvent(key, listener)
 end
 
 function SingleEventManager:connectPropertyChange(key, listener)
-	local event = self._instance:GetPropertyChangedSignal(key)
+	local success, event = pcall(function()
+		return self._instance:GetPropertyChangedSignal(key)
+	end)
+
+	if not success then
+		error(("Cannot get property '%s' changed signal from %s"):format(tostring(key), self._instance.ClassName), 0)
```

Comment:
I think the grammar here might be a bit awkward? What do you think about:
```suggestion
		error(("Cannot subscribe to changed signal: %q is not a property of %s"):format(tostring(key), self._instance.ClassName), 0)
```